doc/README-2.0

```diff
@@ -162,7 +158,7 @@ Usage: pkitool [options...] [common-name]
 Options:
   --batch    : batch mode (default)
   --keysize  : Set keysize
-      size   : size (default=1024)
+      size   : size (default=4096)
```

Comment:
I'll do a bit more research into internals of EasyRSA 2 to see if we could split the key sizes easily.